package com.project.saw.exception;

public class ExceptionMessage {
    public static final String DUPLICATE_EVENT_ERROR_MESSAGE = "This event %s already exist";

    public static final String EVEN_NOT_FOUND_ERROR_MESSAGE = "Event not found with id: ";

    public static final String DUPLICATE_USER_ERROR_MESSAGE = "This user %s already exist";

    public static final String EMAIL_EXISTS_ERROR_MESSAGE = "There is an account with that email adress: ";

}
